client/src/pages/dashboard/chapters/[id]/new_event.tsx

```diff
@@ -0,0 +1,2 @@
+import { NewEventPage } from '../../../../modules/dashboard/Events';
+export default NewEventPage;
```

Comment:
With this `dashboard/events/new` and `dashboard/chapters/1/new_event` are equivalent. I'm not sure having two urls that're essentially the same is the way to go.

What do you think to getting rid of `dashboard/events/new`?

Reply:
Right, `dashboard/events/new` with some _arbitrary_ default chapter is a bit obsolete, when chapter can be picked up from url.